holoviews/plotting/bokeh/util.py

```diff
@@ -239,7 +239,8 @@ def compute_static_patch(document, models):
             update_types[obj['type']].append(key)
     events = [delete_refs(e, IGNORED_MODELS)
               for _, e in sorted(events, key=lambda x: x[0])]
-    events = [e for e in events if all(i in requested_updates for i in get_ids(e))]
+    events = [e for e in events if all(i in requested_updates for i in get_ids(e))
+              if 'new' in e]
```

Comment:
'new' is the key containing the updated value. Not sure why there are events without a 'new' value in the first place since they have no effect anyway.